packages/babel-plugin-transform-parameters/test/fixtures/parameters/arrow-rest-super-no-classes-plugin/options.json

```diff
@@ -0,0 +1,4 @@
+{
+    "plugins": ["transform-parameters"],
+    "throws": "It's not possible to compile `super()` in an arrow function with default or rest parameters without compiling classes.\nPlease add '@babel/plugin-transform-classes' to your Babel configuration."
```

Comment:
This error message is way better than `Unable to handle nested super() usage in arrow`, we can improve that message if we figure out how to make that work.